iib/workers/tasks/build.py

```diff
@@ -496,6 +497,65 @@ def _opm_index_add(
 
     with set_registry_token(overwrite_from_index_token, from_index):
         run_cmd(cmd, {'cwd': base_dir}, exc_msg='Failed to add the bundles to the index image')
+    
+    # add properties to bundles in the index
+    if add_properties:
```

Comment:
How is `add_properties` ever not `false`? If it's set externally, maybe the comment should make clear how?

Reply:
oh nvm, I saw it was passed in. Just needs :param to doc it

Reply:
the idea is just to make that an option in the api so that whatever job runs this command can set that as true.